sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationMockTests.cs

```diff
@@ -707,6 +707,40 @@ public void AnalyzeOperationAnalyzeSentimentWithTwoActions()
 
         #endregion Analyze sentiment
 
+        #region Extract summary
+
+        [Test]
+        public void AnalyzeOperationExtractsSummaryWithTwoActions()
```

Comment:
We have 4 types of mock tests for each type of action:
- DisableServiceLogs
- Options
- Options full
- Test with two actions

`DisableServiceLogs` is not supported currently. We also don't have a corresponding options type. This leaves us with only the two actions test for now.